.github/release.yml

```diff
@@ -0,0 +1,22 @@
+# release.yml
+
+changelog:
+  exclude:
+    labels:
+      - dependencies
+      - chore
+    authors:
+      - dependabot
+  categories:
+    - title: 🚧 Breaking Changes
+      labels:
+        - breaking-change
+    - title: ✨ Enhancements
+      labels:
+        - enhancement
+    - title: 🐛 Bug Fixes
+      labels:
+        - bug
```

Comment:
Nitpick: This should be `bugfix` and not `bug`. I think `bug` is appropriate for issues and `bugfix` is appropriate for PRs.